corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -791,7 +792,7 @@ def append_update(dict_, new_dict):
             datum_ids = {d.datum.id: d for d in datums}
             index = 0
             changed_ids_by_case_tag = defaultdict(list)
-            for this_datum_meta, parent_datum_meta in list(izip_longest(datums, parent_datums)):
+            for this_datum_meta, parent_datum_meta in list(zip_longest(datums, parent_datums)):
```

Comment:
nit: this `list()` call is unnecessary in an iterative context

Reply:
I think it's necessary because of line 803: `datums.insert(...`

Reply:
😱